commcare_connect/program/migrations/0001_initial.py

```diff
@@ -31,9 +32,41 @@ class Migration(migrations.Migration):
                     "delivery_type",
                     models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to="opportunity.deliverytype"),
                 ),
+                (
+                    "organization",
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.DO_NOTHING,
+                        to="organization.organization",
+                    ),
+                ),
+            ],
+            options={
+                "abstract": False,
+            },
+        ),
+        migrations.CreateModel(
+            name="ManagedOpportunity",
```

Comment:
I thought we had decided to add all these to the normal org model? I am ok having them here if we have a good reason for it, I was just surprsied to see it.